packages/core/src/providers/webauthn.ts

```diff
@@ -155,6 +155,8 @@ export interface WebAuthnConfig extends CommonProviderOptions {
     generateAuthenticationOptions: typeof generateAuthenticationOptions
     generateRegistrationOptions: typeof generateRegistrationOptions
   }
+  /** Custom label for the WebAuthn input field */
+  fieldLabel?: string
```

Comment:
I think this is not necessary. https://github.com/nextauthjs/next-auth/blob/e13ffb0cb8355071f84f0292639f2bb7b92976dd/packages/core/src/providers/webauthn.ts#L116

Which similar to Credentials: https://github.com/nextauthjs/next-auth/blob/234a150e2cac3bc62a9162fa00ed7cb16a105244/packages/core/src/providers/credentials.ts#L10-L13

Reply:
Ah good eye! This is perfect